controllers/users.js

```diff
@@ -0,0 +1,59 @@
+import models from '../models/';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+const User = models.User;
+
+dotenv.config();
+
+class Users {
+    static async createUser(req, res) {
+        const {id, displayName, emails, provider} = req.user;
+        try {
+            const existingUser = await User.findOne({where: {"provider": `${provider}: ${id}`}});
```

Comment:
After destructuring provider variable, no need to access it in template literal, you can access variable directly.